### PR TITLE
cherry-pick policy for alpha features

### PR DESCRIPTION
**Pull request by aojea**
Follow up on https://github.com/kubernetes/community/pull/6027 on Jordan's comment https://github.com/kubernetes/community/pull/6027#discussion_r711614882 about backports for alpha features, making it extensive to beta features too.

### CODE REVIEW

**Comment by aojea** [comment]
/assign @liggitt @dims 
I think we should establish the policy for beta too, maybe too strict for beta?

File: contributors/devel/sig-release/cherry-picks.md [hunk 1]
```diff
@@ -42,10 +42,8 @@ Again, the emphasis is on critical bug fixes, e.g.,
 - Panic, crash, hang
 - Security
 
-As a counter example, we would not support backporting a bug fix to an alpha
-feature in general. One condition where we would consider is if that bug caused
-an issue for in another part of the system even when the alpha feature was
-switched off / not in use.
+A bugfix for a functional issue (not a data loss or security issue) that only
+affects an alpha or beta feature does not qualify as a critical bug fix.
```
Comment (liggitt) [inline contributors/devel/sig-release/cherry-picks.md:46]: I think this should be constrained to alpha features for now. The characterization of beta features is:

> The software is well tested. Enabling a feature is considered safe. Features are enabled by default.

I would expect backported bugfixes to beta features to be held to the same standard as any backport (fixes a severe issue, must not introduce risk to the rest of the system, etc), but I wouldn't entirely rule them out.

Comment (aojea) [reply contributors/devel/sig-release/cherry-picks.md:46]: :+1:

**Pull request merged**